db/migrate/20220804165221_add_veteran_icn_to_appeals.rb

```diff
@@ -0,0 +1,7 @@
+class AddVeteranIcnToAppeals < ActiveRecord::Migration[6.1]
+  def change
+    add_column :appeals_api_higher_level_reviews, :veteran_icn, :string, null: true
+    add_column :appeals_api_notice_of_disagreements, :veteran_icn, :string, null: true
+    add_column :appeals_api_supplemental_claims, :veteran_icn, :string, null: true
+  end
+end
```

Comment:
You'll need to commit the updated `schema.rb` file as well, to allow CI and the deploy process to know that there's a migration to run.